src/scripts/main.js

```diff
@@ -2,7 +2,16 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
```

Comment:
```suggestion
const table = document.querySelector('.dashboard tbody');
```
For semantic and styling tables have `thead`, `tbody` and `tfoot` elements. It would be better if you add you rows inside `tbody`, because now your styling seems to be not the same as on example

Reply:
Wow! I even didn't notice it! I'll fix it immediately!